doc/configuration.rst

```diff
@@ -221,7 +221,7 @@ This could be done from the command line, for example:
 
 .. code-block:: console
 
-    $ make html SPHINXOPTS='-D sphinx_gallery_conf.run_stale_examples=True' -Dsphinx_gallery_conf.filename_pattern='my_example_name'``
+    $ make html SPHINXOPTS='-D sphinx_gallery_conf.run_stale_examples=True' -D sphinx_gallery_conf.filename_pattern='my_example_name'
```

Comment:
Also why is the first opt in quotes but not the second...?

Reply:
the whole thing should probably be in the single quotes. And space after the D or not shouldn't matter so if it looks better with a space that's fine